CCN Lite: Add "no 8-Bit support" to documentation

I tried one CCN Lite example. While compiling I got many errors. I fixed some of them, until I reached the following point:
```
ccnl-pkt-ccntlv.c:508:5: error: left shift count >= width of type [-Werror]
```
ccntlv line 508
```
    if ( (unsigned int)len >= ((1 << 16) - sizeof(struct ccnx_tlvhdr_ccnx2015_s)))
        return -1;
```

Since CCN Lite shifts 16 bits, I guess 8-Bit MCUs are not supported. I recommend to add this somewhere to the documentation. :) There are also other places where CCN Lite shifts 16 bits.

--- a/pkg/ccn-lite/Makefile
+++ b/pkg/ccn-lite/Makefile
@@ -1,6 +1,6 @@
 PKG_NAME=ccn-lite
 PKG_URL=https://github.com/cn-uofbasel/ccn-lite/
-PKG_VERSION=b9e22be23ab534015061b03c39348445a2e9bb4b
+PKG_VERSION=e37b02c5cb20e9acccea2394be40f7e570a66a4b
 PKG_LICENSE=ISC
 
 .PHONY: all ..cmake_version_supported

